test/hca-rjsf/config/vaFacility.unit.spec.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { findDOMNode } from 'react-dom';
+import { expect } from 'chai';
+import sinon from 'sinon';
+import ReactTestUtils from 'react-addons-test-utils';
+
+import { DefinitionTester, submitForm } from '../../util/schemaform-utils.jsx';
+import formConfig from '../../../src/js/hca-rjsf/config/form';
+
+describe('Hca general insurance', () => {
+  const { schema, uiSchema } = formConfig.chapters.insuranceInformation.pages.vaFacility;
+  it('should render', () => {
+    const form = ReactTestUtils.renderIntoDocument(
+      <DefinitionTester
+          schema={schema}
+          definitions={formConfig.defaultDefinitions}
+          uiSchema={uiSchema}/>
+    );
+    const formDOM = findDOMNode(form);
+
+    expect(formDOM.querySelectorAll('input,select').length)
+      .to.equal(5);
+    expect(formDOM.querySelectorAll('select')[1].querySelectorAll('option').length).to.equal(1);
```

Comment:
Comment would be useful, e.g. when no state is selected, medical facility list should be empty(ish)